index.html

```diff
@@ -35,4 +35,97 @@ <h1 class="header-logo-wrapper">
       </div>
     </section>
   </body>
+  <body>
+    <section class="about">
+      <div class="about-container">
+        <h2 class="about-title">
+          What is
+          <span class="about-title-Ground"> Ground </span>
```

Comment:
```suggestion
          <span class="about-title-grand"> Grand </span>
```
ground → grandに修正をお願いします。
また、クラス名はケバブケースに統一するようにしましょう。
参考：https://www.wakuwakubank.com/posts/804-it-naming-convention/